frontend/src/components/input.module.css

```diff
@@ -1,54 +1,142 @@
 .inputContainer {
   box-sizing: border-box;
-  display: inline-block;
-  margin: 1rem 0 2rem 0;
+  display: flex;
+  align-items: flex-start;
   position: relative;
   width: 50%;
 }
 
-.input {
-  background-color: var(--border-primary);
-  border-radius: 8px;
-  border: none;
+.textarea {
+  flex: 1;
+  border-radius: 24px;
+  border: 1px solid var(--grey-500);
   box-sizing: border-box;
-  color: var(--text-color-primary);
-  font-size: 1rem;
-  height: 3rem;
+  font-family: 'Roboto', sans-serif;
+  background-color: var(--grey-100);
+  color: var(--grey-900);
+  font-size: 16px;
   outline: none;
-  padding: 0 34px 0 10px;
-  width: 100%;
+  resize: none;
+  padding: 12px 76px 12px 24px;
+  min-height: 48px;
+  max-height: 110px;
+  overflow-y: hidden;
 }
 
-.input:focus {
-  border: 1px solid var(--blue);
+.textarea:focus {
+  background-color: var(--grey-50);
+  border: 2px solid var(--grey-800);
 }
 
-.sendButton {
+.textarea:hover {
+  background-color: var(--grey-200);
+}
+
+.textarea::-webkit-scrollbar {
+  width: 4px;
+}
+
+.textarea::-webkit-scrollbar-thumb {
+  background-color: rgba(0, 0, 0, 0.5);
+  border-radius: 24px;
+}
+
+.textarea::-webkit-scrollbar-thumb:hover {
+  cursor: pointer;
+}
+
+.textarea::-webkit-scrollbar-track {
+  background-color: rgba(0, 0, 0, 0.2);
+  border-radius: 24px;
+}
+
+textarea::placeholder {
+  color: var(--grey-700);
+  font-size: 16px;
+  line-height: 22px;
+  font-family: 'Roboto', sans-serif;
+}
+
+.uploadButton_container {
   background-color: transparent;
+  border-radius: 50%;
+  right: 64px;
+  top: 6px;
+  height: 36px;
+  width: 36px;
+  margin-right: 16px;
+  position: absolute;
+}
+
+.uploadButton {
+  background: transparent;
+  color: var(--grey-900);
   border: none;
   cursor: pointer;
-  margin: 0;
-  outline: none;
-  padding: 0;
   position: absolute;
+  height: 36px;
+  width: 36px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50%;
+}
+
+.uploadButton img {
+  width: 24px;
+  height: 24px;
+}
+
+.uploadButton:hover {
+  background-color: var(--grey-300);
+}
+
+.uploadButton:active {
+  background-color: var(--grey-400);
+}
+
+.uploadButton_container.disabled {
```

Comment:
I think you could potentially change this to `.uploadButton_container:has(.uploadButton:disabled) {` selector then no need to add the `disabled` class to the container and button in JS.